nvflare/app_opt/xgboost/histogram_based_v2/adaptor_executor.py

```diff
@@ -37,15 +38,18 @@ def __init__(
 
         Args:
             adaptor_component_id: the component ID of client target adaptor
+            sender_id: The sender component id
             configure_task_name: name of the config task
             start_task_name: name of the start task
         """
         Executor.__init__(self)
         self.adaptor_component_id = adaptor_component_id
+        self.sender_id = sender_id
```

Comment:
Should validate it to make sure it's either None or a str. You can use functions in validation_utils to do it.

Reply:
Done